vnext/ReactWindowsCore/utilities.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
I am aware that there is already a Utils.h file under ReactWindowsCore. However, looking at Utils.h, I feel that it should be renamed into Url.h. Any thoughts?